.github/workflows/train.yml

```diff
@@ -86,7 +86,7 @@ jobs:
           MLFLOW_TRACKING_URI: http://35.185.111.8:5000
         with:
           args: |
-            compute ssh --zone us-east1-b compute-1 -- "bash /root/train.sh ${MLFLOW_TRACKING_URI} master ${GITHUB_SHA}
```

Comment:
This need to change back?

Reply:
Good catch.

In general, it would be nice to pick up current branch (master or the one PR is coming from) dynamically, but it's a bit complicated and I figured we could skip that for now and set to master